src/routes/+layout.svelte

```diff
@@ -1,9 +1,36 @@
 <script>
+  import { onMount } from 'svelte'
+
   import '../app.css'
   import Footer from '$lib/components/Footer.svelte'
   import Nav from '$lib/components/Nav.svelte'
+  import { slide } from 'svelte/transition'
+
+  let prevScrollPos = 0
+  let showNav = true
+
+  function handleScroll() {
+    const currentScrollPos = window.pageYOffset
+    showNav = prevScrollPos > currentScrollPos
+    prevScrollPos = currentScrollPos
+  }
+
+  onMount(() => {
+    document.title = 'HackHarvard 2023'
+    if (typeof window !== 'undefined') {
```

Comment:
1. Please do not bind event listeners like this in Svelte. Use `on:scroll` for the [`svelte:document` component](https://svelte.dev/docs/special-elements#svelte-document).
2. Even if you need to bind an event listener in Vanilla JS, this is not correct. You should always bind to `document` because the case is rare when you need the extra data in `window` for events. Instead, optimally use `document.addEventListener`.